File: Javascript/level 2/Create Minimal.js
function getMinSum(A,B){
	var answer = 0;
  A.sort(function(a, b) { 
    return a - b;
	});
  B.sort(function(a, b) { 
    return a - b;
	});
  B.reverse();
  for(var i=0; i<A.length;i++)
  	answer+=A[i]*B[i];
	return answer;
}

//아래 코드는 테스트를 위한 출력 코드 입니다.
var tA = [1,2],
	tB = [3,4];

console.log(getMinSum(tA,tB));

File: Python/level 2/Create Minimal.py
def getMinSum(A,B):
    answer = 0
    A.sort()
    B.sort()
    B.reverse()
    for i,j in zip(A,B):
        answer+=i*j
    return answer

#아래 코드는 출력을 위한 테스트 코드입니다.

print(getMinSum([1,2],[3,4]))